apps/i18n/aichat/en_us.json

```diff
@@ -15,6 +15,10 @@
   "chatEventDescriptions_clearChat": "The user cleared the chat workspace.",
   "chatEventDescriptions_loadLevel": "The user loaded the aichat level.",
   "chatEventDescriptions_waitForResponse": "Waiting for response",
+  "chatMessage_showMessage": "Show message",
+  "chatMessage_hideMessage": "Hide message",
+  "chatMessage_flagAsInappropriate": "Flag message as inappropriate",
+  "chatMessage_hasBeenFlagged": "This message has been flagged",
```

Comment:
Update to "This message has been flagged as inappropriate" ?

Reply:
I'm hesitant to add "as inappropriate" since that'll make the string quite long in the UI, and there is context around it to indicate what the flagging is for (from the other untoggled flags) I'd rather wait and if it comes up as confusing we can update.